sal_interfaces/HVAC/HVAC_Events.xml

```diff
@@ -1,18 +1,45 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
-  <Enumeration>SummaryState_DisabledState,SummaryState_EnabledState,SummaryState_FaultState,SummaryState_OfflineState,SummaryState_StandbyState</Enumeration>
-  <Enumeration>DetailedState_DisabledState,DetailedState_EnabledState,DetailedState_FaultState,DetailedState_OfflineState,DetailedState_StandbyState</Enumeration>
-  <SALEvent>
-    <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_logevent_detailedState</EFDB_Topic>
-    <item>
-      <EFDB_Name>detailedState</EFDB_Name>
-      <Description>Detailed state</Description>
-      <!-- Enumeration: DetailedState -->
-      <IDL_Type>long</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALEvent>
+  <Enumeration>
+    DeviceId_chiller01P01=0X0065,
+    DeviceId_chiller02P01=0X0066,
+    DeviceId_chiller03P01=0X0067,
+    DeviceId_crack01P02=0X00C9,
+    DeviceId_crack02P02=0X00CA,
+    DeviceId_fancoil01P02=0X012D,
+    DeviceId_fancoil02P02=0X012E,
+    DeviceId_fancoil03P02=0X012F,
+    DeviceId_fancoil04P02=0X0130,
+    DeviceId_fancoil05P02=0X0131,
+    DeviceId_fancoil06P02=0X0132,
+    DeviceId_fancoil07P02=0X0133,
+    DeviceId_fancoil08P02=0X0134,
+    DeviceId_fancoil09P02=0X0135,
+    DeviceId_fancoil10P02=0X0136,
+    DeviceId_fancoil11P02=0X0137,
+    DeviceId_fancoil12P02=0X0138,
+    DeviceId_manejadoraLower01P05=0X0191,
+    DeviceId_manejadoraLower02P05=0X0192,
+    DeviceId_manejadoraLower03P05=0X0193,
+    DeviceId_manejadoraLower04P05=0X0194,
+    DeviceId_manejadoraSblancaP04=0X01F5,
+    DeviceId_manejadoraSlimpiaP04=0X01F6,
+    DeviceId_vea01P05=0X0259,
+    DeviceId_vea08P05=0X025A,
+    DeviceId_vea09P05=0X025B,
+    DeviceId_vea10P05=0X025C,
+    DeviceId_vea11P05=0X025D,
+    DeviceId_vea12P05=0X025E,
+    DeviceId_vea13P05=0X025F,
+    DeviceId_vea14P05=0X0260,
+    DeviceId_vea15P05=0X0261,
+    DeviceId_vea16P05=0X0262,
+    DeviceId_vea17P05=0X0263,
+    DeviceId_vea01P01=0X02BD,
+    DeviceId_vec01P01=0X02BE,
+    DeviceId_vin01P01=0X02BF,
+    DeviceId_vex03LowerP04=0X0321,
+    DeviceId_vex04CargaP04=0X0322
```

Comment:
I gently suggest using decimal instead of hex so users know that this is not a bitmask.

Reply:
Of course. Sorry, this slipped by my attention and was a left over from a previous iteration. Done.